plugins/modules/setup.yml

```
@@ -58,3 +58,7 @@ EXAMPLES: |
   - name: Gather all facts with a custom timeout on Windows host
     ansible.windows.setup:
       gather_timeout: 30
+
+  - name: Gather Windows facts including custom local facts from C:\CustomFacts
+    ansible.windows.setup:
+      fact_path: 'C:\\CustomFacts'
```

You only need to double up on `\` if you have it enclosed in double quotes. Personally I think the quoting isn't needed here so can just make this simpler.

```suggestion
      fact_path: C:\CustomFacts
```